systemtest/src/test/java/io/strimzi/systemtest/bridge/HttpBridgeST.java

```diff
@@ -259,9 +259,11 @@ void testScaleBridgeSubresource() {
     void testHostAliases() {
         String bridgeName = "bridge-with-hosts";
 
+        String aliasIp = "34.89.152.196";
+        String aliasHostname = "strimzi";
         HostAlias hostAlias = new HostAliasBuilder()
-            .withIp("34.89.152.196")
-            .withHostnames("strimzi")
+            .withIp(aliasIp)
+            .withHostnames(aliasHostname)
             .build();
```

Comment:
Should these be in some abstract class or somewhere once instead of being in any file?